pkg/podvolume/restorer.go

```diff
@@ -213,3 +220,27 @@ func newPodVolumeRestore(restore *velerov1api.Restore, pod *corev1api.Pod, backu
 	}
 	return pvr
 }
+
+func getVolumesRepositoryType(volumes map[string]VolumeBackupInfo) (string, error) {
+	if len(volumes) == 0 {
+		return "", errors.New("empty volume list")
+	}
+
+	// the podVolumeBackups list come from one backup. In one backup, it is impossible that volumes are
+	// backed up by different uploaders or to different repositories. Asserting this ensures one repo only,
+	// which will simplify the following logics
+	repositoryType := ""
+	for _, backupInfo := range volumes {
+		if backupInfo.RepositoryType == "" {
+			return "", errors.New("invalid repository type among volumes")
```

Comment:
This is actually empty, why not make the error msg like `empty repository type`, also this is an error for one particular volume, therefore would it be helpful to put more info in the error to help identify the volume?

Reply:
Done, changed the message to `"empty repository type found among volume snapshots, snapshot ID %s, uploader %s"`